releasenotes/notes/telemetry-logs-validate-unique-messages-5a020f0531952e0c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Telemetry logs: telemetry writer validates unique messages for each interval.
```

Comment:
```suggestion
    Telemetry logs: telemetry writer validates unique messages for each interval.

```

Reply:
thoughts on this? I don't know if it's a rule but I feel like we always keep the section before the `:` as one word? And added a newline